apps/src/applab/designElements/ColorPickerPropertyRow.jsx

```diff
@@ -28,7 +28,7 @@ var ColorPickerPropertyRow = React.createClass({
    * Make our button a colpick color picker, if it isn't already
    */
   ensureColorPicker: function () {
-    var element = React.findDOMNode(this.refs.colorPicker);
+    var element = ReactDOM.findDOMNode(this.refs.colorPicker);
```

Comment:
Ah!  My change of all 12 instance broke tests, and I figured out why: [Refs to custom (user-defined) components work exactly as before; only the built-in DOM components are affected by this change.](https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#dom-node-refs)

So this particular case, where `refs.colorPicker` is a `<button>` we can replace this, but in some cases (like `refs.age` in ShareWarnings.jsx) we still have to use findDOMNode because the ref is on a user-defined component.

Fixing...